packages/flutter_tools/test/general.shard/resident_runner_test.dart

```diff
@@ -1196,6 +1196,48 @@ void main() {
     }) async => mockVMService,
   }));
 
+
+  // Don't use this outside of the test below.
```

Comment:
I know you don't like groups, but should we wrap this to enforce not re-using this?

Reply:
Yeah, good point - this is the right place to use one

Reply:
Done

Reply:
LGTM